data/policies/nested_alias.cas

```diff
@@ -0,0 +1,22 @@
+virtual resource tmp {
+	// All children must implement read
+	virtual fn read(domain source) {}
+}
+
+@associate([tmp])
+virtual domain foo {}
+
+domain bar inherits foo {
+    // Policies must contain at least one AV rule
+	allow(this, resource, file, [read]);
+
+    @alias(zap)
+	extend tmp {}
+}
+
+domain abc {
+    resource xyz {}
+
+    @alias(bob)
```

Comment:
Can we add rules to this actually referencing zap and bob so that the test proves out using them as well?